<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="alarm_name_label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="alarm_name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="parameters_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="if_layout">
     <item row="0" column="3">
      <widget class="QComboBox" name="countable_combo_box"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="test_type_combo_box"/>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="behavior_combo_box"/>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="count_spin_box"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="if_label">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>IF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="in_layout">
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="in_label">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>IN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="zone_combo_box">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="active_times_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Active Times</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTimeEdit" name="start_time_edit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="to_label">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="stop_time_edit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>alarm_name</tabstop>
  <tabstop>test_type_combo_box</tabstop>
  <tabstop>count_spin_box</tabstop>
  <tabstop>countable_combo_box</tabstop>
  <tabstop>behavior_combo_box</tabstop>
  <tabstop>zone_combo_box</tabstop>
  <tabstop>start_time_edit</tabstop>
  <tabstop>stop_time_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_button_box</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_button_box</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
